<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerbrauchTextuellReport</class>
 <widget class="QWidget" name="VerbrauchTextuellReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verbrauch</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_useTillDate">
         <property name="text">
          <string>Daten ber√ºcksichtigen &amp;nur bis zum </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_till">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_all">
         <property name="text">
          <string>&amp;Alles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_umsatz">
         <property name="text">
          <string>nur &amp;Umsatz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_aufwand">
         <property name="text">
          <string>nur Auf&amp;wand</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showTableCode">
         <property name="text">
          <string>&amp;Tischbereiche anzeigen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_includeLMData">
         <property name="text">
          <string>&amp;LM Daten</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_export">
         <property name="text">
          <string>&amp;Exportieren</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_period"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ReportTextViewWidget" name="textView" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReportTextViewWidget</class>
   <extends>QWidget</extends>
   <header>reports.reportTextViewWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
